src/includes/performance/traces-sampler-as-filter/ruby.mdx

```diff
@@ -0,0 +1,38 @@
+```ruby
+Sentry.init do |config|
+  config.traces_sampler = lambda do |sampling_context|
+    # if this is the continuation of a trace, just use that decision (rate controlled by the caller)
+    unless sampling_context[:parent_sampled].nil?
+      next sampling_context[:parent_sampled]
+    end
+
+    # transaction_context is the transaction object in hash form
+    # keep in mind that sampling happens right after the transaction is initialized
+    # for example, at the beginning of the request
+    transaction_context = sampling_context[:transaction_context]
+
+    # transaction_context helps you sample transactions with more sophistication
+    # for example, you can provide different sample rates based on the operation or name
+    op = transaction_context[:op]
+    transaction_name = transaction_context[:name]
+
+    case op
+    when /request/
+      case transaction_name
+      when /health_check/
+        0.0
+      when /payment/
+        0.5
+      when /api/
+        0.2
+      else
+        0.1
+      end
+    when /sidekiq/
+      0.01 # you may want to set a lower rate for background jobs if the number is large
+    else
+      0.0 # ignore all other transactions
+    end
+  end
+end
+```
```

Comment:
Why are `src/includes/performance/traces-sampler-as-filter/ruby.mdx` and `src/includes/performance/traces-sampler-as-sampler/ruby.mdx` identical?

I'm hesitant to add two files that are exactly the same.

Looking at the examples for other platforms, they are different. The `traces-sampler-as-filter` example focuses on dropping certain transactions when a condition is met. The `traces-sampler-as-sampler` focuses on returning different rates depending on transaction.

Reply:
Because I think the example happen to cover both of the cases.

Reply:
I've simplified this example to make it focus more on filtering.